packages/synapse-react-client/stories/FeaturedDataTabs.stories.ts

```diff
@@ -41,6 +57,16 @@ export const SingleQuery: Story = {
 }
 
 export const QueryPerCard: Story = {
+  parameters: {
+    chromatic: { viewports: [600, 1200] },
+    msw: {
+      handlers: [
+        // In reality, we actually make multiple table queries that return different data
+        // It will suffice if all queries return the same data for this example
+        ...getHandlersForTableQuery(mockQueryPerCardResult, MOCK_REPO_ORIGIN),
```

Comment:
We'd need to somehow map query requests to generated result IDs to support multiple table query handlers at one time (so we would basically need a database of some sort)